cocktails9/cocktails9/ViewModel/CocktailViewModel.swift

```diff
@@ -0,0 +1,130 @@
+import Foundation
+
+import SwiftUI
+import Combine
+
+class CocktailViewModel: ObservableObject {
+    
+    @Published var cocktails: [Cocktail] = []
+    private let cocktailService: CocktailService
+    
+    private var user: User?
+    @Published var activeFilter: (type: FilterType, value: String)?
+    
+    
+    
+    init(cocktailService: CocktailService) {
+        self.cocktailService = cocktailService
+        if let currentUser = UserManagement.shared.getLoggedInUser() {
```

Comment:
Feel free to remove this if let and assign UserManagement.shared.getLoggedInUser() directly to user variable. Both are optionals so end result will be same